python/tests/unit/arcticdb/version_store/test_filtering.py

```diff
@@ -892,7 +892,7 @@ def test_filter_numeric_isin_hashing_overflows(lmdb_version_store, df_col, isin_
     assert_frame_equal(expected, result)
 
 
-def test_filter_numeric_isin_unsigned(lmdb_version_store):
+def test_filter_numeric_isin_unsigned_not_hypothesis(lmdb_version_store):
```

Comment:
Calling a test `not_hypothesis` doesn't describe what the test actually _does_. Every test in the codebase could have that suffix.

This test must be testing a specific case seperately to the hypothesis test. What is that case?

Reply:
because it is forcing the column to be uint64 (2**64 - 1 is one of the values), I would name it as *_uint64 or *_max_uint64?

Reply:
Both are better than `not_hypothesis`.

Reply:
changed it